terraform/api_gateway.tf.json.template.py

```diff
@@ -300,13 +300,13 @@ def for_domain(cls, domain):
                                     'none': {}
                                 },
                                 'visibility_config': {
-                                    'metric_name': 'AWS-CommonRuleSet',
+                                    'metric_name': 'aws_common_rule_set',
```

Comment:
and this are the same string. Try to use a comprehension over a singleton to make this more dry.

With that technique, `{{…, 'foo': 'bar', …}}` becomes `*[{…, 'foo': name, …} for name in ['bar']]`. If want to propose an alternative, please propose it in PL before implementing it. Because this coupling is so local, adding a config attribute would be overkill.